src/libaktualizr/primary/sotauptaneclient.h

```diff
@@ -139,14 +157,14 @@ class SotaUptaneClient {
   Uptane::ImagesRepository images_repo;
   Uptane::Manifest uptane_manifest;
   std::shared_ptr<INvStorage> storage;
-  std::shared_ptr<PackageManagerInterface> package_manager_;
   std::shared_ptr<HttpInterface> http;
+  std::shared_ptr<PackageManagerInterface> package_manager;
```

Comment:
Why remove the final underscore? It's a small point, but I thought we wanted to actually move to all member variables having the final underscore, which IIRC is recommended by the Google style guide.

Reply:
Wanted to make it more consistent with other members, but you are right, we probably should move to the opposite direction. Reverted.